{{cookiecutter.repo_name}}/requirements/production.txt

```diff
@@ -1,7 +1,10 @@
 # Packages required in a production environment
 -r base.txt
 
-gevent==1.0.2
-gunicorn==19.3.0
-MySQL-python==1.2.5
+gevent==1.1rc1
```

Comment:
Honestly, I'm happy to leave this PR open until I've taken the hit to make sure that course-discovery is stable using the rc (or the released version, if we get there). We also need an unreleased version of gunicorn to make python3 work.